addons/actions/src/lib/util/getPropertiesList.js

```diff
@@ -0,0 +1,18 @@
+const { hasOwnProperty } = Object.prototype;
+
+export default function getPropertiesList(value) {
+  const keys = [];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const name in value) {
+    // noinspection JSUnfilteredForInLoop
+    if (
+      keys.indexOf(name) === -1 &&
```

Comment:
Why is this check needed? There's no chance that `for .. in` yields duplicate keys:
```js
const a = {foo: 'foo', bar: 'bar'}
const b = Object.create(a, {foo: '{value: 'baz', enumerable: 'true'}})
for (key in b) { console.log(key) }
VM372:1 foo
VM372:1 bar
```
Note that `foo` is logged only once

Reply:
Sorry, a left over from populating the keys array with ownPropertyNames first, I will remove it.

The function check is needed to only show functions in case they are own properties of the current object.

The previous fix only worked because the results were stringified, filtering out all functions e.g. `for .. in` over a file object **does** include `slice` etc. but were not shown, because of the stringification later on.